photoeditor/src/main/java/ja/burhanrashid52/photoeditor/DrawingView.kt

```diff
@@ -216,6 +209,9 @@ class DrawingView @JvmOverloads constructor(
 
     // region constructors
     init {
-        setupBrushDrawing()
+        //Caution: This line is to disable hardware acceleration to make eraser feature work properly
```

Comment:
I had to move it from `setupBrushDrawing()` directly to `init` to fix compilation error `Property must be initialized or be abstract`